src/views/Events/index.js

```diff
@@ -187,22 +169,19 @@ export default class Events extends Component {
   clearFilters() {
     this.setState(
       {
-        includePast: false,
+        includePast: true,
         chapelCredits: false,
-        academics: false,
         admissions: false,
         art: false,
         sports: false,
-        calendar: false,
         cec: false,
-        fair: false,
         chapelOffice: false,
         studentLife: false,
       },
       async () => {
-        const events = await gordonEvent.getFilteredEvents(this.state);
-        this.setState({ filteredEvents: events, loading: false });
-        this.createURLParameters();
+        // Set includePast to true above so that it is "toggled" to false by the below method
+        // This will ensure we filter only future events
+        await this.togglePastEvents();
```

Comment:
It's true that this method is not at all clear in it's scope. Ideally, this would be fixed with a refactoring of the component, but that is beyond the scope of this pr. I'll revisit it if I have time.